add astika.is-a.dev

astika327-dev opened the pull request:
<!--
YOU MUST FILL OUT THIS TEMPLATE ENTIRELY FOR YOUR PR TO BE ACCEPTED, NONE OF IT IS OPTIONAL UNLESS SPECIFIED.
-->

# Requirements
<!-- Your domain MUST pass ALL the requirements below, otherwise it WILL BE DENIED. -->

<!-- Change each checkbox to [x] (all lowercase, with no spaces between the brackets) to mark it as completed. -->

- [x ] I have **read** and **agree** to the [Terms of Service](https://is-a.dev/terms). <!-- Your request MUST follow the TOS to be approved. -->
- [x ] My file is following the [domain structure](https://docs.is-a.dev/domain-structure/).
- [x ] My website is **reachable** and **completed**. <!-- We do not permit simple "Hello, world!" or simply copied/mostly blank templated websites. -->
- [x ] My website is **software development** related. <!-- Only your root subdomain needs to meet this requirement. -->
- [x ] My website is not for commercial use. <!-- Your website's purpose should not be to generate any form of revenue. -->
- [ x] I have provided sufficient contact information in the `owner` key. <!-- Provide your email in the `email` field or another platform (e.g. X/Twitter or Discord) for contact. -->
- [ x] I have provided a preview of my website below. <!-- This step is required for your domain to be approved. -->

# Website Preview
<!-- Provide a link or screenshot of your website below. -->
Website Preview: https://astika327-dev.github.io/portofolio/

astika327-dev committed:
Create astika.json

astika327-dev committed:
Rename astika.json to delete.json

astika327-dev committed:
Create astika.json

astika327-dev committed:
Merge branch 'main' into main

STICKnoLOGIC commented on delete.json:
you mean, literally "delete" it? you can safely delete this file

STICKnoLOGIC commented on domains/astika.json:
```diff
@@ -0,0 +1,11 @@
+    "username": "astika327-dev",
+    "email": "astika327@gmail.com"
+  },
+  "record": {
```
records, with **"S"**
```suggestion
  "records": {
```

STICKnoLOGIC reviewed (changes requested):
- incomplete website, fix the button of each project
- resolve issues below

astika327-dev committed:
Delete delete.json

astika327-dev committed:
Update astika.json

fix: change record to records and correct json

astika327-dev commented:
✅ Deleted the `delete.json` file  
✅ Updated `astika.json` (changed `record` → `records` as suggested)

Thank you for the review, please check if everything looks correct now.

STICKnoLOGIC reviewed (approved):
lgtm, wait for maintainers to review and merge pr
Edit: site is not yet fixed

STICKnoLOGIC reviewed (changes requested):
fix the site

astika327-dev commented:
Approved.

On Sat, Sep 6, 2025, 12:44 AM John Aerial J. Azcune <
***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> fix the site
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/is-a-dev/register/pull/25921#pullrequestreview-3190089182>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BV7MDZDFBABYWKYHRCZ6J7T3RG4WLAVCNFSM6AAAAACFWTTP42VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZTCOJQGA4DSMJYGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

astika327-dev committed:
Merge branch 'main' into main

astika327-dev committed:
Merge branch 'main' into main

astika327-dev committed:
Merge branch 'main' into main

astika327-dev commented:
Haha, thanks!

On Sat, Sep 6, 2025, 10:44 PM John Aerial J. Azcune <
***@***.***> wrote:

> *STICKnoLOGIC* left a comment (is-a-dev/register#25921)
> <https://github.com/is-a-dev/register/pull/25921#issuecomment-3262319158>
>
> Approved, i'll fix the site asap
>
> even i approve it, it will screened by the maintainers and they will ask
> the same as what I ask :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/is-a-dev/register/pull/25921#issuecomment-3262319158>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BV7MDZCLIU3QNI2YFGV7VSL3RLXMLAVCNFSM6AAAAACFWTTP42VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTENRSGMYTSMJVHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

astika327-dev committed:
Merge branch 'main' into main

astika327-dev commented:
Thanks for reviewing. I’ve updated the structure and will continue improving the portfolio:
Cleaned up the codebase to make it easier to maintain.
Improved the resume section with concise, point-based highlights.
Planning to polish project pages with better descriptions and visuals (no more placeholders).
Will refine the contact form so it actually works for email/WhatsApp.
Feedback is welcome, especially on design choices and overall flow.

astika327-dev committed:
Merge branch 'main' into main

STICKnoLOGIC reviewed (approved):
lgtm
